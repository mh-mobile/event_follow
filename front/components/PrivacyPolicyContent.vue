<template lang="pug">
  div
    h2 プライバシーポリシー
    p EVENT FOLLOWは（以下「当サイト」といいます。）は、当サイトの提供するサービス（以下「本サービス」といいます。）における、ユーザーについての個人情報を含む利用者情報の取扱いについて、以下のとおりプライバシーポリシー（以下「本ポリシー」といいます。）を定めます。
    h3 1. 収集する利用者情報及び収集方法
    p 本ポリシーにおいて、「利用者情報」とは、ユーザーの識別に係る情報、通信サービス上の行動履歴、その他ユーザーまたはユーザーの端末に関連して生成または蓄積された情報であって、本ポリシーに基づき当サイトが収集するものを意味するものとします。
    p 本サービスにおいて当サイトが収集する利用者情報は、その収集方法に応じて、以下のようなものとなります。
    ol.bracket-number
      li ユーザーが本サービスの利用において、他のサービスと連携を許可することにより、当該他のサービスからご提供いただく情報 
        ul
          li 当該外部サービスでユーザーが利用するID 
          li その他当該外部サービスのプライバシー設定によりユーザーが連携先に開示を認めた情報 
      li ユーザーが本サービスを利用するにあたって、当サイトが収集する情報 
        ul
          li リファラ
          li IPアドレス
          li サーバーアクセスログに関する情報
          li Cookie、その他の識別子

    h3 2.	利用目的
    p 本サービスのサービス提供にかかわる利用者情報の具体的な利用目的は以下のとおりです。
    ol.bracket-number
      li 本サービスに関する登録の受付、本人確認、ユーザー認証、ユーザー設定の記録、本サービスの提供、維持、保護及び改善のため
      li ユーザーのトラフィック測定及び行動測定のため
      li 本サービスに関するご案内、お問い合わせ等への対応のため
      li 本サービスに関する当サイトの規約、ポリシー等（以下「規約等」といいます。）に違反する行為に対する対応のため
      li 本サービスに関する規約等の変更などを通知するため
    h3 3. 通知・公表または同意取得の方法、利用中止要請の方法
    p ユーザーは、本サービスの所定の設定を行うことにより、利用者情報の全部または一部についてその収集又は利用の停止を求めることができ、この場合、当サイトは速やかに、当サイトの定めるところに従い、その利用を停止します。なお利用者情報の項目によっては、その収集または利用が本サービスの前提となるため、当サイト所定の方法により本サービスを退会した場合に限り、当サイトはその収集又は利用を停止します。
    h3 4. プライバシーポリシーの変更手続
    p 当サイトは、必要に応じて、本ポリシーを変更します。但し、法令上ユーザーの同意が必要となるような本ポリシーの変更を行う場合、変更後の本ポリシーは、当サイト所定の方法で変更に同意したユーザーに対してのみ適用されるものとします。なお、当サイトは、本ポリシーを変更する場合には、変更後の本ポリシーの施行時期及び内容を当サイト上での表示その他の適切な方法により周知し、またはユーザーに通知します。
    br
    p 【2021年1月29日制定】
</template>

<style lang="scss" scoped>
h1 {
  font-size: 2em;
  margin: 0.67em 0;
  font-weight: bold;
}

h2 {
  font-size: 1.5em;
  margin: 0.75em 0;
  font-weight: bold;
}

h3 {
  font-size: 1.17em;
  margin: 0.83em 0;
  font-weight: bold;
}

ol {
  padding: 0;
  margin: 20px;
  li {
    padding-left: 1em;
    line-height: 1.5;
    padding-top: 5px;
  }
}

ol.bracket-number {
  padding: 0;
  margin: 20px;

  > li {
    padding-left: 1em;
    line-height: 1.5;
    padding-top: 5px;
    list-style-type: none;
    counter-increment: cnt;

    ul {
      padding: 0;
      margin: 20px;
      li {
        list-style-type: disc;
        padding-left: 1em;
        line-height: 1.5;
        padding-top: 5px;
      }
    }
  }

  > li::before {
    display: marker;
    content: "(" counter(cnt) ") ";
  }
}

ul {
  padding: 0;
  margin: 20px;
  li {
    list-style-type: disc;
    padding-left: 1em;
    line-height: 1.5;
    padding-top: 5px;
  }
}
</style>
