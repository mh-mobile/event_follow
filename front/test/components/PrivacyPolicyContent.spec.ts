import { createLocalVue, shallowMount } from "@vue/test-utils"
import VueCompositionApi from "@vue/composition-api"
import PrivacyPolicyContent from "@/components/PrivacyPolicyContent.vue"

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
describe("PrivacyPolicyContent.vue", () => {
  it("renders html correctly", () => {
    const wrapper = shallowMount(PrivacyPolicyContent, {
      localVue
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
