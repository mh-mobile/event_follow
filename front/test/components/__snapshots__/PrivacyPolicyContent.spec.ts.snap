// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPolicyContent.vue renders html correctly 1`] = `
"<div>
  <h2>プライバシーポリシー</h2>
  <p>EVENT FOLLOWは（以下「当サイト」といいます。）は、当サイトの提供するサービス（以下「本サービス」といいます。）における、ユーザーについての個人情報を含む利用者情報の取扱いについて、以下のとおりプライバシーポリシー（以下「本ポリシー」といいます。）を定めます。</p>
  <h3>1. 収集する利用者情報及び収集方法</h3>
  <p>本ポリシーにおいて、「利用者情報」とは、ユーザーの識別に係る情報、通信サービス上の行動履歴、その他ユーザーまたはユーザーの端末に関連して生成または蓄積された情報であって、本ポリシーに基づき当サイトが収集するものを意味するものとします。</p>
  <p>本サービスにおいて当サイトが収集する利用者情報は、その収集方法に応じて、以下のようなものとなります。</p>
  <ol class=\\"bracket-number\\">
    <li>ユーザーが本サービスの利用において、他のサービスと連携を許可することにより、当該他のサービスからご提供いただく情報 <ul>
        <li>当該外部サービスでユーザーが利用するID </li>
        <li>その他当該外部サービスのプライバシー設定によりユーザーが連携先に開示を認めた情報 </li>
      </ul>
    </li>
    <li>ユーザーが本サービスを利用するにあたって、当サイトが収集する情報 <ul>
        <li>リファラ</li>
        <li>IPアドレス</li>
        <li>サーバーアクセスログに関する情報</li>
        <li>Cookie、その他の識別子</li>
      </ul>
    </li>
  </ol>
  <h3>2. 利用目的</h3>
  <p>本サービスのサービス提供にかかわる利用者情報の具体的な利用目的は以下のとおりです。</p>
  <ol class=\\"bracket-number\\">
    <li>本サービスに関する登録の受付、本人確認、ユーザー認証、ユーザー設定の記録、本サービスの提供、維持、保護及び改善のため</li>
    <li>ユーザーのトラフィック測定及び行動測定のため</li>
    <li>本サービスに関するご案内、お問い合わせ等への対応のため</li>
    <li>本サービスに関する当サイトの規約、ポリシー等（以下「規約等」といいます。）に違反する行為に対する対応のため</li>
    <li>本サービスに関する規約等の変更などを通知するため</li>
  </ol>
  <h3>3. 通知・公表または同意取得の方法、利用中止要請の方法</h3>
  <p>ユーザーは、本サービスの所定の設定を行うことにより、利用者情報の全部または一部についてその収集又は利用の停止を求めることができ、この場合、当サイトは速やかに、当サイトの定めるところに従い、その利用を停止します。なお利用者情報の項目によっては、その収集または利用が本サービスの前提となるため、当サイト所定の方法により本サービスを退会した場合に限り、当サイトはその収集又は利用を停止します。</p>
  <h3>4. プライバシーポリシーの変更手続</h3>
  <p>当サイトは、必要に応じて、本ポリシーを変更します。但し、法令上ユーザーの同意が必要となるような本ポリシーの変更を行う場合、変更後の本ポリシーは、当サイト所定の方法で変更に同意したユーザーに対してのみ適用されるものとします。なお、当サイトは、本ポリシーを変更する場合には、変更後の本ポリシーの施行時期及び内容を当サイト上での表示その他の適切な方法により周知し、またはユーザーに通知します。</p><br>
  <p>2021年1月29日制定</p>
</div>"
`;
